.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
+    "root": true,
+    "env": {
+        "browser": true,
+        "es2020": true
+    },
+    "plugins": [
+        "@typescript-eslint",
+        "eslint-plugin-import"
+    ],
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "prettier"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "tsconfig.json"
+    },
+    "rules": {
+        "no-misleading-character-class": "warn",
+        "no-dupe-else-if": "warn",
+        "no-warning-comments": "warn",
+        "import/no-absolute-path": "warn",
+        
+        "@typescript-eslint/no-explicit-any": "off",        
+        "@typescript-eslint/ban-ts-comment": "off",      
+        "@typescript-eslint/ban-types": "off",      
+        "@typescript-eslint/no-unused-vars": "off"
```

Comment:
I guess these rules are just temporary?

Reply:
Yeah, need to remove them after solving all unnecessary conditions warnings.